compiler/dyno/include/chpl/framework/error-classes-list.h

```diff
@@ -38,6 +38,18 @@ ERROR_CLASS(PrototypeInclude, const uast::Include*, const uast::Module*)
 ERROR_CLASS(MissingInclude, const uast::Include*, std::string)
 ERROR_CLASS(Redefinition, const uast::NamedDecl*, std::vector<ID>)
 ERROR_CLASS(AmbiguousConfigName, std::string, const uast::Variable*, ID)
-ERROR_CLASS(AmbiguousConfigSet, const uast::Variable*, std::string, std::string)
+ERROR_CLASS(AmbiguousConfigSet,
+    const uast::Variable*,
+    std::string,
+    std::string)
 WARNING_CLASS(ImplicitSubModule, const uast::Module*, UniqueString)
-WARNING_CLASS(ImplicitFileModule, const uast::AstNode*, const uast::Module*, const uast::Module*)
+WARNING_CLASS(ImplicitFileModule,
+    const uast::AstNode*,
+    const uast::Module*,
+    const uast::Module*)
+ERROR_CLASS(ProcTypeUnannotatedFormal,
```

Comment:
I've been deliberately putting the warnings below the errors, but I don't know if that's good ordering. What are your thoughts?

Reply:
I don't think an ordering mattered and would rather just do FCFS, but it's up to you I guess

Reply:
Keep it as is then :)